MANIFEST.in

```diff
@@ -17,5 +17,8 @@ include 3rdparty/pybind11/CMakeLists.txt
 include 3rdparty/lexertl14/CMakeLists.txt
 recursive-include 3rdparty/lexertl14/include/lexertl/ *
 
-recursive-include 3rdparty/pybind11/tools/ *
-include CMakeLists.txt
```

Comment:
confused how it works w/o the `CMakeLists.txt` file, but sdist is adding it 'magically'; let's make it explicit.

line 14/15 need to go